app/_src/policies/external-services.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: External Service
 ---
+{% if_version gte:2.8.x %}
+New to Kuma? You don't need this, check [`MeshExternalService`](/docs/{{ page.version }}/policies/meshexternalservice) instead.
```

Comment:
so, is it going GA? Does policy matching work?

Reply:
AFAIK thanks to @lukidzi policy matching works if you're using egress (so the same way it used to work so people are not losing any features). I'll do examples of this.

Reply:
It works for MeshTrafficPermisison

Reply:
Correction: It's only MTP so it can't be here.

Reply:
Added a note that the resource is experimental (the same way MeshService is).